src/engine/keyManager.js

```diff
@@ -290,32 +290,7 @@ export class KeyManager {
   }
 
   async signMessage ({ message, address }: SignMessage) {
```

Comment:
Is the `signMessage` method even used anymore? Can we remove the whole thing?

Reply:
I think someone is using it.
I'm not 100% but I do remember we created it for a reason.
Keep it here for now and mention it at the next engineering meeting.